src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlob.java

```diff
@@ -171,6 +172,7 @@
      * Stores the last modification timestamp.
      */
     public static final Mapping LAST_MODIFIED = Mapping.named("lastModified");
+    @DefaultValue("")
     private LocalDateTime lastModified = LocalDateTime.now();
```

Comment:
shouldn't we always skip this for LocalDate & LocalDateTime properties? using the given default makes almost no sense anywhere (?)

Reply:
If we want to do this we can overwrite the column method in the properties classes, can copy from MultiLanguageStringProperty

Reply:
Is now fixed with https://github.com/scireum/sirius-db/pull/449 as suggested